pkg/alertmanager/amcfg.go

```diff
@@ -1673,51 +1672,51 @@ func (gc *globalConfig) sanitize(amVersion semver.Version, logger log.Logger) er
 	if gc.SlackAPIURLFile != "" {
 		if gc.SlackAPIURL != nil {
 			msg := "'slack_api_url' and 'slack_api_url_file' are mutually exclusive - 'slack_api_url' has taken precedence"
-			level.Warn(logger).Log("msg", msg)
+			logger.Warn(msg)
 			gc.SlackAPIURLFile = ""
 		}
 
 		if amVersion.LT(semver.MustParse("0.22.0")) {
 			msg := "'slack_api_url_file' supported in Alertmanager >= 0.22.0 only - dropping field from provided config"
-			level.Warn(logger).Log("msg", msg, "current_version", amVersion.String())
+			logger.Warn(msg, "current_version", amVersion.String())
 			gc.SlackAPIURLFile = ""
 		}
 	}
 
 	if gc.OpsGenieAPIKeyFile != "" && amVersion.LT(semver.MustParse("0.24.0")) {
 		msg := "'opsgenie_api_key_file' supported in Alertmanager >= 0.24.0 only - dropping field from provided config"
-		level.Warn(logger).Log("msg", msg, "current_version", amVersion.String())
+		logger.Warn(msg, "current_version", amVersion.String())
 		gc.OpsGenieAPIKeyFile = ""
 	}
 
 	if gc.SMTPAuthPasswordFile != "" && amVersion.LT(semver.MustParse("0.25.0")) {
 		msg := "'smtp_auth_password_file' supported in Alertmanager >= 0.25.0 only - dropping field from provided config"
-		level.Warn(logger).Log("msg", msg, "current_version", amVersion.String())
+		logger.Warn(msg, "current_version", amVersion.String())
 		gc.SMTPAuthPasswordFile = ""
 	}
 
 	if gc.SMTPAuthPassword != "" && gc.SMTPAuthPasswordFile != "" {
 		msg := "'smtp_auth_password' and 'smtp_auth_password_file' are mutually exclusive - 'smtp_auth_password' has taken precedence"
-		level.Warn(logger).Log("msg", msg)
+		logger.Warn(msg)
 		gc.SMTPAuthPasswordFile = ""
 	}
 
 	if gc.VictorOpsAPIKeyFile != "" && amVersion.LT(semver.MustParse("0.25.0")) {
 		msg := "'victorops_api_key_file' supported in Alertmanager >= 0.25.0 only - dropping field from provided config"
```

Comment:
Ha, so weird that we declare a string variable to use it only once 😅 

Well, not scope for this PR

Reply:
Yep, I am not changed it, but it is odd